docs/sphinx/source/whatsnew/v0.9.1.rst

```diff
@@ -88,3 +89,4 @@ Contributors
 * Will Hobbs (:ghuser:`williamhobbs`)
 * Uday Guntupalli (:ghuser:`uguntupalli`)
 * Shashwata Roy (:ghuser:`RoyCoding8`)
+* Shashwata Roy (:ghuser: `RoyCoding8`)
```

Comment:
I don't think this second entry is needed, since it just repeats the line immediately before it. Also I don't think the extra space is necessary.

Reply:
It should be moved to v0.9.2.rst anyway since we've already released v0.9.1 :)

Reply:
Sure, I will move and remove the duplicate entry.